analysis_templates/cms_minimal/law.cfg

```diff
@@ -49,8 +49,8 @@ default_keep_reduced_events: True
 # slightly to the left to avoid them being excluded from the last bin; None leads to automatic mode
 default_histogram_last_edge_inclusive: None
 
-# boolean flag that, if True, sets the *hists* output of cf.SelectEvents and cf.MergeSelectionStats to optional
-default_selection_hists_optional: True
+# boolean flag that, if True, configures cf.SelectEvents to create statistics histograms
+default_create_selection_hists: False
```

Comment:
I do not like setting this parameter to False per default, because this means that histograms will suddenly not be produced anymore, even if users might already be expected to have them produced. Not sure, how many users already use them, if my analysis is the only one, then I would not mind

Reply:
Right, the default should be True :+1: